esx_service/vmdk_ops.py

```diff
@@ -644,7 +645,7 @@ def listVMDK(tenant):
     Each volume name is returned as either `volume@datastore`, or just `volume`
     for volumes on vm_datastore
     """
-    vmdk_utils.init_datastoreCache()
+    vmdk_utils.init_datastoreCache(force=True)
```

Comment:
I think it need to make sure the ```listVMDK``` return the correct value even if the "datastore" name is changed outside. Cache need to be forced refresh to pickup the new datastore name.